packages/tasks/src/tasks/placeholder/spec/input.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Is `placeholder` intended to be committed?

Reply:
Yes, in case we ever have to add a new task to this package